source-browser.html

```diff
@@ -72,16 +109,16 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           name: topic,
           repoUrl: `https://github.com/${ORGANIZATION}/apertium-${topic}`,
         }));
-
+        
```

Comment:
Please remove _all_ these extra trailing spaces.

Reply:
These are not fixed...

Reply:
I have removed these from throughout the code.